code-analysis/crates/solidity/outputs/documentation/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+# how to do the build.rs without an actual crate?
```

Comment:
I suggest deleting that crate, if we are not doing that migration now.

Reply:
But we still need it as a place to run the `build.rs`, which doesn't belong anywhere else.